compiler/test/dotty/tools/scripting/ClasspathTests.scala

```diff
@@ -51,7 +51,7 @@ class ClasspathTests:
       // convert scriptCp to a list of files
       val hashbangJars: List[File] = scriptCp.split(psep).map { _.toFile }.toList
       val hashbangClasspathJars = hashbangJars.map { _.name }.sorted.distinct // get jar basenames, remove duplicates
-      val packlibDir = s"$scriptCwd/$packLibDir" // classpathReport.sc specifies a wildcard classpath in this directory
+      val packlibDir: String = ??? /* ??? was s"$scriptCwd/$packLibDir" */ // classpathReport.sc specifies a wildcard classpath in this directory
```

Comment:
Erm ... how does this not crash?

Reply:
this is in an ignored test, but I deleted that method, it wasn't worth rewriting an ignored test